docs/makedoc.sh

```diff
@@ -40,4 +40,5 @@ docker run --rm \
   pseudomuto/protoc-gen-doc --doc_opt=markdown,docs.md
 
 sphinx-multiversion . ${BUILD_DIR} -b dirhtml
-mv -v _build/dirhtml/${LATEST_JINA_VERSION}/* _build/dirhtml
+mv -v ${BUILD_DIR}/${LATEST_JINA_VERSION}/* _build/dirhtml
+cp ${BUILD_DIR}/404/index.html ${BUILD_DIR}/404.html
```

Comment:
If we move 404.html, I think we'll need to handle asset urls too. 

```suggestion
cp ${BUILD_DIR}/404/index.html ${BUILD_DIR}/404.html
sed -i 's/href="\.\./href="/' 404.html 
```